### PR TITLE
Bug/144 route url

### PR DESCRIPTION
**Pull request by mohitb3**
The simplest fix I could find for the route bugs https://github.com/USACE/water-ui/issues/144 was to remove the createNestedUrlBundle and use the doUpdateUrl directly. 
I also made some styling changes to the homepage and made the navbar toggle-able if you go into mobile mode. If you guys don't like the homepage changes I can change it back, I just wanted to try something new. I also updated the homepage search box placeholder https://github.com/USACE/water-ui/issues/153 and removed the search box from the navbar if you're on the homepage https://github.com/USACE/water-ui/issues/154

### CODE REVIEW

**Comment by brian428** [comment]
Mohit, out of curiosity, do you know how this is working when on localhost? I see you're using pkg.homepage to set the base url, but that is set to "https://apps.rsgis.dev/water". Does something override this somehow when running on localhost?

**Comment by mohitb3** [comment]
Yea so I do 
```
const homepage = new URL(pkg.homepage);
const baseUrl = homepage.pathname;
```
so the baseUrl will just be `/water`, not the whole `https://apps.rsgis.dev/water`. When you run it on local host it'll take you to `http://localhost:3000/water` by default. The createNestedUrlBundle was doing the same thing behind the scenes, but it didn't append `/water` to the anchor tags' href which is why the bug was being caused.

File: src/app-bundles/routes-bundle.js [hunk 1]
```diff
@@ -15,41 +15,39 @@ import AllLocations from "../app-pages/reports/corp-office/office-location/all-l
...
+// get the base url for which all the routes will be nested under
+const homepage = new URL(pkg.homepage);
+const baseUrl = homepage.pathname;
+
 export const RoutePaths = {
-  Home: `/`,
-  Help: `/help`,
+  Home: baseUrl,
+  Help: `${baseUrl}/help`,
   Locations: `/locations/:locationId`,
```
Comment (brian428) [inline src/app-bundles/routes-bundle.js]: Next question: is there a reason Locations is the only route that doesn't use ${baseUrl}? Or was this just an oversight?

Comment (mohitb3) [reply src/app-bundles/routes-bundle.js]: haha woops this was an oversight. I'll push up a fix

**Review by dlin2102 — APPROVED** [review]

**Comment by brian428** [comment]
Pushed some tweaks, including switching back to using that background on the homepage. I think it was too hard to read without the background color (at least with the image we are using...we can revisit if we pick a less "noisy" background image). https://github.com/USACE/water-ui/pull/158/commits/11e012e10aa7f4ee4d985c9207b8aba1bf3511ae

**Pull request merged**